src/Github.hs

```diff
@@ -152,6 +153,7 @@ instance FromJSON PullRequestPayload where
     <*> getNested v ["pull_request", "head", "sha"]
     <*> getNested v ["pull_request", "title"]
     <*> getNested v ["pull_request", "user", "login"]
+    <*> (v .: "pull_request" >>= (\v1 -> v1 .:? "body" .!= ""))
```

Comment:
According to Github's [own docs](https://docs.github.com/en/webhooks/webhook-events-and-payloads?actionType=opened#pull_request), the `pull_request.body` key will always be present. It will just be `null` if the PR description was empty.

This combined with the change of `body` being `Maybe Text`, and I believe you can use `getNested` like the other cases.